lang/cs/Org.Apache.REEF.Wake/Remote/Impl/ObserverContainer.cs

```diff
@@ -120,6 +122,8 @@ public void OnError(Exception error)
 
         public void OnCompleted()
         {
+            Logger.Log(Level.Info, "Exiting the Observer Container");
+
```

Comment:
Remove the empty line. Or the changes to this file completely, as the change is orthogonal to the goals of this PR.